requirements-test.txt

```diff
@@ -1,2 +1,4 @@
+coverage==5.*
+coveralls==1.*
```

Comment:
Missed it, when looked on this first time, but I would hold `coveralls==3.*`. I hope it'll be integrated with coveralls.io better.

Reply:
I guess it is due to [dropping of Python 2 support](https://github.com/TheKevJames/coveralls-python/blob/master/CHANGELOG.md#200-2020-04-07), but now we run it only for 3.8.

Reply:
updated
```
--- a/requirements-test.txt
+++ b/requirements-test.txt
@@ -1,4 +1,4 @@
 coverage==5.*
-coveralls==1.*
+coveralls==3.*
 flake8==3.7.9
 hypothesis==4.*
```

Reply:
Moved coveralls package installation to GH actions workflow

Reply:
From CI on Python 2:

> ERROR: Could not find a version that satisfies the requirement coveralls==3.* (from -r requirements-test.txt (line 2)) (from versions: 0.1, 0.1.1, 0.2, 0.3, 0.4, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.5, 1.0a1, 1.0a2, 1.0b1, 1.0, 1.1, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.8.0, 1.8.1, 1.8.2, 1.9.0, 1.9.1, 1.9.2, 1.10.0, 1.11.0, 1.11.1)
> ERROR: No matching distribution found for coveralls==3.* (from -r requirements-test.txt (line 2))

I guess [environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) should do the work:

```
coveralls==3.*; python_version > '3.4'
```

Reply:
> Moved coveralls package installation to GH actions workflow

Or this way, okay.